WeatherMood/ViewController/EmoticonSelectingViewController.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  EmoticonSelectingViewController.swift
+//  WeatherMood
+//
+//  Created by 이지원 on 2021/07/23.
+//
+
+import UIKit
+
+final class EmoticonSelectingViewController: BaseViewController {
```

Comment:
```suggestion
final class SelectingEmoticonViewController: BaseViewController {
```

Selecting을 앞에쓰는게 낫지않을까요? 
`WritingDiaryViewController`처럼요.

Reply:
현수님이 그려놓으신 프로젝트 구조 이름 그대로 생성한 거 였는데...ㅎㅎ 수정했습니다!